Simulation/Get_FSM_State.m

```diff
@@ -38,6 +41,6 @@
       endif
     elseif last_state == 3 %if the plane has crashed
       %the plane cannot change states;
-      last_state = 3
+      next_state = 3
```

Comment:
Unrelated to this PR but we should really make states an [enum](https://www.mathworks.com/help/matlab/matlab_oop/enumerations.html).

Reply:
I'll make a quick issue about it